designs/tinkerbell-provider-input.md

```diff
@@ -0,0 +1,358 @@
+# Required configuration for EKS Anywhere Tinkerbell provider
+
+## Introduction
+
+The purpose of this document is to explain the different input files required to create an EKS Anywhere Tinkerbell cluster and how to generate them.
+
+### Goals and objectives
+
+As an EKS Anywhere Tinkerbell provider user, I want to:
+
+* Generate the required cluster configuration file with minimal customizations
+* Generate the required hardware files using CSV files that contain hardware information
+
+### State of Scope
+
+**In Scope**
```

Comment:
Are we separating the scope into two parts? I recall us talking about starting off with these files as input and then moving towards only specifying the id's with all the hardware information embedded in the tinkerbell side.

Reply:
Those details aren't finalized yet, so I don't know what this will to look like post-beta. I would prefer to keep this doc just for beta and maybe update/add a new one once we have full knowledge of what we want for GA.

Reply:
Made the scope specific to just beta as post-beta details are still tentative.